tests/textnote_tests.ts

```diff
@@ -140,7 +140,7 @@ function superscriptAndSubscript(options: TestOptions): void {
   const voice2 = score.voice(notes2);
   voice2.getTickables().forEach((note) => {
     const textNote = note as TextNote;
-    textNote.setFont({ family: Font.SERIF, size: 15 });
+    textNote.setFont({ family: Tables.lookupMetric('fontFamily'), size: 15 });
```

Comment:
I would avoid using Tables.lookupMetric(...) in our *_tests.ts files.

I treat the tests as one way to teach developers how to use the public API. So if we do this here, we are declaring that Tables.lookupMetric('fontFamily') is a public API that should be relied on.

I'd rather just put a string here, like 'serif'.

If we do not specify the fontFamily, won't it default to the right thing?

Can this be written as:

```
 textNote.setFont({ size: 15 });
```

Reply:
I changed it to setFontSize